supervisor/bootstrap.py

```diff
@@ -319,6 +321,25 @@ def filter_data(event, hint):
             }
         )
 
+        # Sanitize event
+        for tag in event.get("tags", []):
+            if "url" in tag:
+                tag[1] = sanitize_url(tag[1])
+
+        if event.get("request"):
+            if event["request"].get("url"):
+                event["request"]["url"] = sanitize_url(event["request"]["url"])
+
+            for header in event["request"].get("headers", []):
+                if header[0] == "Referer":
+                    header[1] = sanitize_url(header[1])
+
+                if header[0] == "X-Hassio-Key":
```

Comment:
All kind of headers which we use are in `const.py`. There are more around

Reply:
Good idea